src/test/packages/nginx/refs/zarf.yaml

```diff
@@ -0,0 +1,28 @@
+kind: ZarfPackageConfig
+metadata:
+  name: nginx
+  version: 0.0.2
+
+components:
+  - name: nginx-remote
+    required: true
+    manifests:
+      - name: simple-nginx-deployment
+        namespace: nginx
+        files:
+          - deployment.yaml
+    actions:
```

Comment:
2 suggestions:
- I think we can kill this boilerplate action
- add a description to the `metadata` saying why this pkg exists

Reply:
Disregard that first suggestion, it's probably important that we ensure the pods actually start up if we're referring to different refs